utils/email/index.js

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import fs from 'fs';
+import sendgridEmail from '@sendgrid/mail';
+import mjml2html from "mjml";
+
+if (process.env.SENDGRID_API_KEY) {
+  sendgridEmail.setApiKey(process.env.SENDGRID_API_KEY);
+}
+
+
+function sendEmail({recipient, subject, htmlBody, textBody}) {
+  if (!process.env.SENDGRID_API_KEY) {
+    console.error('SendGrid error: SENDGRID_API_KEY is not set');
+    return;
+  }
+  const from = 'mentors@tupu.io'; // Change to your verified sender
+  recipient = process.env.DEV_EMAIL_RECIPIENT || recipient;
+  subject = buildSubjectLine(subject);
+  const msg = {
+    to: recipient,
+    from: process.env.DEV_EMAIL_RECIPIENT || from, 
```

Comment:
Does it make sense to also replace the from? Or did you have delivery issues with `mentors@tupu.io`?

Reply:
I did not try with `mentors@tupu.io`. 

I don't use the tupu's sendgrid, I use my own key (and so an external collaborator can do).

When I registered my SendGrid profile, it made me verify the sender, and the quick start suggested to use the verified address as sender. I assumed that it would work only with a verified sender.

I will give it a try and remove this if it works.

Reply:
It works only with verified senders. I have then added a `SENDGRID_EMAIL_VERIFIED_SENDER` in the `.env.local` so that 
1. **on a dev machine** we can set both `SENDGRID_EMAIL_VERIFIED_SENDER` and `DEV_EMAIL_RECIPIENT` to a dev email (personal sendgrid account, verified)
2. **on netlify** you can set  `SENDGRID_EMAIL_VERIFIED_SENDER` to the verified one for tupu, and temporarily `DEV_EMAIL_RECIPIENT` to yours

Reply:
@tsg  please add `SENDGRID_EMAIL_VERIFIED_SENDER` on the netlify `.env.local`

Reply:
Got it, thanks!

I have verified the domain, so I think any sender that is `@tupu.io` should work. Creating a re-deploy to test it.